plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -4,17 +4,17 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"github.com/olivere/elastic"
 	"math"
 	"net/http"
+	"net/url"
 	"strconv"
 	"strings"
 	"text/template"
 	"time"
 
 	"crypto/sha256"
 
-	"gopkg.in/olivere/elastic.v5"
-
 	"github.com/influxdata/telegraf"
```

Comment:
Please move the `"github.com/olivere/elastic"` import to where the line `"gopkg.in/olivere/elastic.v5" was before. We usually group the imports into "standard libraries" (e.g. fmt, net/url etc), "3rd party libraries" (like the elastic one) and "telegraf internal libraries" (everything stating with `github.com/influxdata/telegraf`).